packages/theming/atlas/src/themesource/atlas_ui_resources/web/core/widgets/_navigationlist.scss

```diff
@@ -16,7 +16,7 @@
 
     li.mx-navigationlist-item {
         @include transition();
-        padding: $grid-padding-top $grid-padding-right $grid-padding-bottom $grid-padding-left;
+        padding: $spacing-medium $spacing-medium;
```

Comment:
if both are same, lets apply only `padding: $spacing-medium;`

Reply:
updating to just $spacing-medium